### PR TITLE
Fix safex_acccount new not catching missing params

### PR DESCRIPTION
**Pull request by ulrich9595**
Function
Safex_account new <username>

Expected Result
Error saying that account data is required.  Otherwise there's no indication that the command was used incorrectly and the account does not appear in the safex_account list.

Result
 No output whatsover. 

How to replicate
Run the command safex_account new <username> without any account data.

Fix
The safex_account new requires two parameters, so added a check to make sure there are at least two parameters passed in.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -75,6 +75,12 @@ dist-static:
 	cd build/dist && cmake -D STATIC=ON -D Protobuf_USE_STATIC_LIBS=ON -D BUILD_SAFEX_PROTOBUF_RPC=ON -D ARCH="x86-64" -D BUILD_64=ON -D CMAKE_BUILD_TYPE=release -D BUILD_WALLET_RPC=ON -D BUILD_ADVANCED_WALLET=ON ../.. && $(MAKE)
 
 
+dist-static-protobuf:
+	mkdir -p build/dist-protobuf
+	cd build/dist-protobuf && cmake -D STATIC=ON -D Protobuf_USE_STATIC_LIBS=ON -D BUILD_SAFEX_PROTOBUF_RPC=ON -D ARCH="x86-64" -D BUILD_64=ON -D CMAKE_BUILD_TYPE=release -D BUILD_WALLET_RPC=ON -D BUILD_ADVANCED_WALLET=ON ../.. && $(MAKE)
+
+	
+
```
Comment (ulrich9595) [inline Makefile]: I did not explicitly make this change in my branch, so not sure where this came from.  This change probably should not be merged.

File: src/cryptonote_core/CMakeLists.txt [hunk 2]
```diff
@@ -29,7 +29,8 @@
 # Parts of this file are originally copyright (c) 2014-2018 The Monero Project
 
 
-if(BUILD_SAFEX_PROTOBUF_RPC)    
+if(BUILD_SAFEX_PROTOBUF_RPC)
+
```
Comment (ulrich9595) [inline src/cryptonote_core/CMakeLists.txt]: this looks benign, but I also didn't not explicitly make this change.  not sure where it came from.

File: src/simplewallet/simplewallet_safex.cpp [hunk 3]
```diff
@@ -856,8 +856,12 @@ namespace cryptonote
...
     if (command == "new")
     {
-      const std::string &username = local_args[0];
+      if (local_args.size() != 2){
+        fail_msg_writer() << tr("usage: safex_account new <account_username> <account_data>");        
+        return true;
+      }
 
+      const std::string &username = local_args[0];
       std::ostringstream accdata_ostr;
```
Comment (ulrich9595) [inline src/simplewallet/simplewallet_safex.cpp]: this is the only file I changed.

**Comment by VanGrx** [comment]
There are a lot of not-needed changes inside this pull request. As I already created a fix for this, I think it will be easier to push it from my repository.

**Pull request closed without merging**